### ISSUE TITLE
Hyper not loading

### ISSUE THREAD

**Issue by cdock1029** [carloscuesta/hyper-materialshell#9]
Hello @carloscuesta

- Hyper Version 1.2.1 (1.2.1.1527)

Here is my `.hyper.js` file (just installed new copy of hyper so this is stock file):

```javascript
module.exports = {
  config: {
    // default font size in pixels for all tabs
    fontSize: 15,

    // font family with optional fallbacks
    fontFamily: 'FiraCode-Retina,SourceCodePro-Medium, Menlo,"Lucida Console",monospace',

    // terminal cursor background color and opacity (hex, rgb, hsl, hsv, hwb or cmyk)
    cursorColor: 'rgba(255,64,129,0.75)',//'#6200EA',//'rgba(248,28,229,0.75)',

    // `BEAM` for |, `UNDERLINE` for _, `BLOCK` for █
    cursorShape: 'BLOCK',

    // color of the text
    foregroundColor: '#fff',

    // terminal background color
    backgroundColor: '#000',

    // border color (window, tabs)
    borderColor: '#333',

    // custom css to embed in the main window
    css: '',

    // custom css to embed in the terminal window
    termCSS: '',

    // custom padding (css format, i.e.: `top right bottom left`)
    padding: '12px 14px',

    // the full list. if you're going to provide the full color palette,
    // including the 6 x 6 color cubes and the grayscale map, just provide
    // an array here instead of a color map object
    colors: {
      black: '#000000',
      red: '#ff0000',
      green: '#33ff00',
      yellow: '#ffff00',
      blue: '#0066ff',
      magenta: '#cc00ff',
      cyan: '#00ffff',
      white: '#d0d0d0',
      lightBlack: '#808080',
      lightRed: '#ff0000',
      lightGreen: '#33ff00',
      lightYellow: '#ffff00',
      lightBlue: '#0066ff',
      lightMagenta: '#cc00ff',
      lightCyan: '#00ffff',
      lightWhite: '#ffffff'
    },

    // the shell to run when spawning a new session (i.e. /usr/local/bin/fish)
    // if left empty, your system's login shell will be used by default
    shell: '/usr/local/bin/zsh'

    // for advanced config flags please refer to https://hyperterm.org/#cfg
  },

  // a list of plugins to fetch and install from npm
  // format: [@org/]project[#version]
  // examples:
  //   `hyperpower`
  //   `@company/project`
  //   `project#1.0.1`
  plugins: [
		'hyper-materialshell'
	],

  // in development, you can create a directory under
  // `~/.hyperterm_plugins/local/` and include it here
  // to load it and avoid it being `npm install`ed
  localPlugins: []
};

```

Trying to open hyper usually results in nothing happening. No terminal window opens. Once, I saw this error:

<img width="532" alt="screen shot 2017-02-26 at 8 49 59 pm" src="https://cloud.githubusercontent.com/assets/802586/23346207/25452890-fc66-11e6-8367-176f452590f8.png">

**Comment by carloscuesta** [comment]
I see, this is because it's not entering inside of the switch. Providing an empty object inside of .hyper.js, should solve the problem.

```javascript
materialshell:{}
```

In the meantime you can use this while I fix this issue 👍

**Comment by carloscuesta** [comment]
Fixed in :bookmark: 1.5.1
